package.json

```diff
@@ -22,17 +35,21 @@
     "sql-formatter": "^15.0.2",
     "swr": "^2.2.4",
     "tailwind-merge": "^2.2.0",
-    "typescript": "^4.4.2",
```

Comment:
Do we need need this? Just curious.

Reply:
Yes, it gives types and compilation optimisations that are useful 😁